### ISSUE TITLE
Nomacs 

### ISSUE THREAD

**Issue by leochras** [microsoft/winget-pkgs#13817]
# Nomacs

Manifest:

```YAML
PackageIdentifier:   # nomacs.nomacs
PackageVersion:      # 3.16.224
PackageLocale:       # en-US
Publisher:           # nomacs
PackageName:         # nomacs
License:             # GPL 3.0
ShortDescription:    # nomacs - Image Lounge
Installers: 
 - Architecture:     # x64
   InstallerType:    # exe/msi/zip
   InstallerUrl:     # https://github.com/nomacs/nomacs/releases/tag/3.16.224
   InstallerSha256:  # SHA256 hash calculated from installer
ManifestType: singleton
ManifestVersion: 1.0.0
```

**Comment by KaranKad** [comment]
I have submitted the pull request #13830, but instead of version 3.16.224 according to their GitHub release, I have added version 3.16.1709 because.
### The app reports itself so
![image](https://user-images.githubusercontent.com/71691514/119242122-228c7780-bb79-11eb-99a3-82dae9d20fae.png)
### And in the settings app
![image](https://user-images.githubusercontent.com/71691514/119242135-3637de00-bb79-11eb-8780-52d2b5581c43.png)

**Comment by leochras** [comment]
Thank you! Should I close this issue?

**Comment by KaranKad** [comment]
Yeah, the app is now available through winget.
![image](https://user-images.githubusercontent.com/71691514/119253517-2fd45100-bbcf-11eb-84ad-ea73a8d92483.png)

**Comment by Dragon1573** [comment]
>  new version of nomacs is available, `3.17.2297`. 

Hi @hginzel, 👋🏼 

I can't find any installer file related to this exact version. There's `v3.16` available but nowhere for `v3.17`. 😕 

- [Windows (7/8/10)](https://nomacs.org/docs/getting-started/installation/#windows-7-8-10) points to the latest stable release on [GitHub Releases](https://github.com/nomacs/nomacs/releases/latest), which is currently `v3.16.224`.
